src/Microsoft.TestPlatform.PlatformAbstractions/net451/Tracing/PlatformEqtTrace.cs

```diff
@@ -29,25 +29,32 @@ public partial class PlatformEqtTrace : IPlatformEqtTrace
         public void SetupRemoteEqtTraceListeners(AppDomain childDomain)
         {
             Debug.Assert(childDomain != null, "domain");
-            if (childDomain != null && !Enum.Equals(TraceLevel, TraceLevel.Off))
+            if (childDomain != null)
             {
                 RemoteEqtTrace remoteEqtTrace = (RemoteEqtTrace)childDomain.CreateInstanceFromAndUnwrap(
                     typeof(RemoteEqtTrace).Assembly.Location,
                     typeof(RemoteEqtTrace).FullName);
```

Comment:
so we are creating a remote trace object in a new appdomain even the trace is off. Seems odd.